.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,128 @@
+name: Publish Docker Image
+
+on:
+  push:
+    branches:
+      - main
+      - v1.16
+    tags:
+      - v1.*
+
+jobs:
+  build:
```

Comment:
I'm not sure, but I think separating `arm` and `amd` jobs might be more maintainable.

Reply:
hmm, it is reasonable. I'll fix it.
Thanks.

Reply:
Thanks!
Please let me know if there is anything I can do.

If separating the jobs,  it would be good to make `echo-x86-images` and `echo-arm64-images` instead of using `echo-all-images` and remove the `case` condition.